README.md

```diff
@@ -1 +1,62 @@
-# annual-diabetes-walk
\ No newline at end of file
+# Annual Diabetes Walk
+
+> This is a website to document the activities of a diabetes organization that aims at raising awareness 
+for individuals living with diabetes and raising funds to support the campaign against diabetes from people 
+all over the country of Kenya.
+
+
+## Built With
+
+- Html
+- Css
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Html
+- Css
+- Javascript
+- Git
+- Github
+
+
+### Install
+1. Go to terminal and ```cd``` into the directory of your preference.
+2.  Run ```git clone git@github.com:alphayowakarindi/annual-diabetes-walk.git``` to clone the app.
+3. ```npm install``` to install the dependencies in the ```package.json``` file for testing linters errors
+
+
+### Run tests
+1. For HTML run ```npx hint .```
+2. For CSS run ```npx stylelint "**/*.{css,scss}"```
+3. For Javascript run ```npx eslint .```
+
+## Live Demo
+
+[Live Demo Link](https://www.loom.com/share/1d25061a596a4864aeeabf48d81e1d52)
+
```

Comment:
- [OPTIONAL] For section `Live Demo`  you are supposed to put deployed live link of the app and for your link that has a project video explanation you can create a new section with similar title you would like for example Video link or etc.